### DISCUSSION TITLE
Font styles not working

### DISCUSSION THREAD

**Discussion by Ananthumpillai** [GrapesJS/grapesjs#5734] [Q&A]
Hi
All fonts in font family doesn't
[screencast-from-09-03-24-074558-pm-ist_4A8bTYPo.webm](https://github.com/GrapesJS/grapesjs/assets/138502626/121b3932-93c1-4ac2-9503-d4ed297d81a2)
 seems working,

**Comment by artf** [comment]
Looks more like an issue with your browser/OS 🤔
